omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -150,6 +150,11 @@
     win_build_env = {
       "PIP_FIND_LINKS" => "#{build_deps_dir}",
     }
+    # Some libraries (looking at you, aerospike-client-python) need EXT_CFLAGS instead of CFLAGS.
+    specific_build_env = {
+      "aerospike" => nix_build_env.merge({"EXT_CFLAGS" => nix_build_env["CFLAGS"] + " -std=gnu99"}),
+    }
```

Comment:
Nit (for later, no need to fix for now since on Windows, we don't build the aerospike wheel manually): I think we should have two of these hashes, one `nix_specific_build_env` where the items extend `nix_build_env`, and is used on Linux / macOS, and another `win_specific_build_env` where the items extend `win_build_env`, and is used on Windows. 

Right now, if a dependency has to be manually installed on Windows, it will use the nix flags as its base, which isn't what we want.

Same for the py3 file.